lib/utils.js

```diff
@@ -0,0 +1,10 @@
+function getTypeIdByName(types, name) {
```

Comment:
nice work implementing the bonus function, for that extra partial credit, would want to see this function itself tested as well